lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -1007,7 +1007,11 @@ DIFile *SPIRVToLLVMDbgTran::getFile(const SPIRVId SourceId) {
          "DebugSource instruction is expected");
   SPIRVWordVec SourceArgs = Source->getArguments();
   assert(SourceArgs.size() == OperandCount && "Invalid number of operands");
-  StringRef Checksum(getString(SourceArgs[TextIdx]));
+  string ChecksumStr =
```

Comment:
```suggestion
  std::string ChecksumStr =
```
I see that 'using namespace std' is used here, but this is considered as a good practice to specify namespace (or at least namespace alias) in general.